cylc/flow/task_remote_cmd.py

```diff
@@ -105,6 +105,7 @@ def remote_init(install_target, rund, *dirs_to_symlink, indirect_comm=None):
         src = os.path.expandvars(val)
         if '$' in src:
             print(REMOTE_INIT_FAILED)
```

Comment:
Should this be `print(REMOTE_INIT_FAILED, file=sys.stderr)`?

Reply:
We parse this text from the scheduler so it all has to go to one file, either stdout or stderr.

Reply:
As it stands, it is returned in proc_ctx.out, which is where we look for the messages from the remote.